client/my-sites/menus/item-options/posts.jsx

```diff
@@ -54,6 +54,7 @@ var Posts = React.createClass( {
 		var posts = this.props.posts.slice();
 		if ( this.props.type === 'page' ) {
 			posts.unshift( siteMenus.generateHomePageMenuItem( this.getHomePageTitle() ) );
+			posts.push( siteMenus.generateNewPageMenuItem() );
```

Comment:
I think it should be bottom but it should be rendered on top of the scrolling view, like:

![image](https://cloud.githubusercontent.com/assets/548849/13780576/48ab1636-eac0-11e5-8cbd-ae309e9e5fdf.png)